### PR TITLE
Fix 5.0 actions pre-70

### PR DESCRIPTION
**Pull request by gr3ger**
~~This is pretty much done, but I'm unsure on one part (which is why this shouldn't be merged yet)
line 388 in app/js/ffxivcraftmodel.js is where the name of/brand of things are handled, but I don't know exactly how much the increase in progress is now since the game doesn't explicitly tell you.~~
_Edit: This is most likely solved now, but please take a look nonetheless so I didn't make some stupid mistake._

### CODE REVIEW

**Comment by gr3ger** [comment]
@doxxx , I would love it if you could help me out with the elemental thing I mentioned in the description. Once that is done we have _mostly_ fixed pre-70 changes.

**Comment by gr3ger** [comment]
From what I can see this is complete now, unless I missed something.
So please take a look, test it and see if it suits your needs :)

File: app/js/services/actions.js [hunk 1]
```diff
@@ -55,19 +54,8 @@
...
-    brandOfWind: {                                       skillID: {"Carpenter": 100006} },
-
-    nameOfEarth: {                           buff: true, skillID: {"Leatherworker": 4571} },
-    nameOfFire: {                            buff: true, skillID: {"Blacksmith": 4569} },
-    nameOfIce: {                             buff: true, skillID: {"Armorer": 4570} },
-    nameOfLightning: {                       buff: true, skillID: {"Weaver": 4572} },
-    nameOfWater: {                           buff: true, skillID: {"Alchemist": 4573} },
-    nameOfWind: {                            buff: true, skillID: {"Carpenter": 4568} },
+    brandOfTheElements: {                                skillID: {"Alchemist": 0, "Armorer": 0, "Blacksmith": 0, "Carpenter": 0, "Culinarian": 0, "Goldsmith": 0, "Leatherworker": 0, "Weaver": 0} },
+    nameOfTheElements: {                     buff: true, skillID: {"Alchemist": 0, "Armorer": 0, "Blacksmith": 0, "Carpenter": 0, "Culinarian": 0, "Goldsmith": 0, "Leatherworker": 0, "Weaver": 0} },
```
Comment (doxxx) [inline app/js/services/actions.js]: Mark these as `common: true`. then you only need one copy of the action icon png files in the `img/actions` folder.

File: app/js/services/actions.js [hunk 2]
```diff
@@ -24,7 +24,7 @@
     standardTouch: {                                     skillID: {"Alchemist": 100093, "Armorer": 100034, "Blacksmith": 100018, "Carpenter": 100004, "Culinarian": 100109, "Goldsmith": 100078, "Leatherworker": 100048, "Weaver": 100064} },
     advancedTouch: {                                     skillID: {"Alchemist": 100097, "Armorer": 100038, "Blacksmith": 100022, "Carpenter": 100008, "Culinarian": 100112, "Goldsmith": 100081, "Leatherworker": 100052, "Weaver": 100068} },
     hastyTouch: {                                        skillID: {"Culinarian": 100108} },
-    byregotsBlessing: {                                  skillID: {"Carpenter": 100009} },
+    byregotsBlessing: {                                  skillID: {"Alchemist": 0, "Armorer": 0, "Blacksmith": 0, "Carpenter": 0, "Culinarian": 0, "Goldsmith": 0, "Leatherworker": 0, "Weaver": 0} },
```
Comment (doxxx) [inline app/js/services/actions.js]: This can also become common.

File: app/js/services/actions.js [hunk 3]
```diff
@@ -34,15 +34,14 @@
     manipulation: {                          buff: true, skillID: {"Goldsmith": 278} },
     innerQuiet: {              common: true, buff: true, skillID: {"Alchemist": 258, "Armorer": 254, "Blacksmith": 253, "Carpenter": 252, "Culinarian": 259, "Goldsmith": 255, "Leatherworker": 257, "Weaver": 256} },
     steadyHand: {              common: true, buff: true, skillID: {"Alchemist": 250, "Armorer": 246, "Blacksmith": 245, "Carpenter": 244, "Culinarian": 251, "Goldsmith": 247, "Leatherworker": 249, "Weaver": 248} },
-    steadyHand2: {                           buff: true, skillID: {"Culinarian": 281} },
+    steadyHand2: {                           buff: true, skillID: {"Alchemist": 0, "Armorer": 0, "Blacksmith": 0, "Carpenter": 0, "Culinarian": 0, "Goldsmith": 0, "Leatherworker": 0, "Weaver": 0} },
```
Comment (doxxx) [inline app/js/services/actions.js]: This can also become common.

**Comment by gr3ger** [comment]
This should be good to go now, unless you found something more :)
Thanks for the input!

**Comment by doxxx** [comment]
Thank you so much!

**Pull request merged**